pycbc/results/pygrb_postprocessing_utils.py

```diff
@@ -527,6 +448,9 @@ def extract_basic_trig_properties(trial_dict, trigs, slide_dict, seg_dict):
             trig_time[slide_id] = numpy.asarray([])
             trig_snr[slide_id] = numpy.asarray([])
         trig_bestnr[slide_id] = trigs['network/reweighted_snr'][indices]
+        trig_bestnr[slide_id][trig_bestnr[slide_id] <
+                              opts.newsnr_threshold] = 0
```

Comment:
And here: please use [this](https://github.com/gwastro/pycbc/pull/4649/threshold_only) function.